questionnaires/templates/survey/survey_landing.html

```diff
@@ -1,13 +1,19 @@
 {% extends "base.html" %}
-{% load wagtailimages_tags %}
-
-{% block content %}
-    <div> thank you page</div>
-
-    <div>{{ page.thank_you_text }}</div>
+{% load static wagtailimages_tags %}
 
+{% block extra_css %}
+    <link rel="stylesheet" type="text/css" href="{% static 'css/survey.css' %}">
+{% endblock %}
 
-    {% if back_url %}
-        <a href="{{ back_url }}">< Back</a>
-    {% endif %}
+{% block content %}
+    <div class="container">
+        <h1 class="title survey-page__title survey-page__title--center">Thank
+            you for your input!</h1>
+        <div class="survey-page__desc"><p>Complete a Covid-19 course</p></div>
```

Comment:
Temp removing hard coded text in order to merge.
```suggestion
        <div class="survey-page__desc"><p></p></div>
```